### DISCUSSION TITLE
Symbiota Support Group Meetings

### DISCUSSION THREAD

**Discussion by themerekat** [Symbiota/community-board#34] [Announcements]
The Symbiota Support Hub invites you to join our [Symbiota Support Group Meetings](https://symbiota.org/symbiota-support-group/)!

They are held the first Monday of every month at 12:00 noon Pacific / 1:00 PM Mountain / 2:00 Central / 3:00 Eastern.

In these meetings, we will demonstrate a Symbiota tool or feature, make important announcements, and answer user questions. The demonstration portion, but not the Q&A, will be recorded. More information about these sessions can be found here: [https://www.idigbio.org/content/symbiota-support-group](https://www.idigbio.org/content/symbiota-support-group).

Have a topic in mind that you'd like covered during one of these meetings? Comment here with ideas!

**Comment by ljwalker** [comment]
From [RMBL](https://swbiodiversity.org/seinet/collections/misc/collprofiles.php?collid=112): keys & character matrices

**Reply by ljwalker** [reply]
This topic has now been covered: https://youtu.be/GMk2WkE3Hno

**Comment by ljwalker** [comment]
From SoRo user: Discussion of collecting event data in Symbiota portals

**Reply by themerekat** [reply]
Can we get more clarification on this? Like, do we store event-based data?

**Reply by ljwalker** [reply]
Another SoRo user adds, "I would love to hear more about managing event data in symbiota, like how can we create events and lump records into them. Or how can we get the data out of symbiota." @ricklevy21

**Comment by ljwalker** [comment]
From CCH2 user: How to update snapshot collections

**Reply by themerekat** [reply]
This has now been covered! https://youtu.be/0i6zudJFtu8

**Comment by jesslane13** [comment]
How about personal observation management?  We've been investigating this as a tool for some of our collectors to enter their own data and make their own labels, without burdening them with full editor access.  I haven't seen much about this in the docs.

**Reply by ljwalker** [reply]
@jesslane13 - This topic has been scheduled for Dec. 5: https://symbiota.org/symbiota-support-group/

**Reply by themerekat** [reply]
Here's the recording! https://youtu.be/u65P_3VqGVI

**Comment by ljwalker** [comment]
Jennifer Girón (@JCGiron) requests a session on glossaries in Symbiota portals.

**Reply by themerekat** [reply]
@JCGiron, you'll be excited to hear that we've added documentation about the glossary to Symbiota Docs! https://biokic.github.io/symbiota-docs/portal_manager/glossary/

**Comment by themerekat** [comment]
We should have a session on making checklists

**Reply by themerekat** [reply]
This has been covered! https://youtu.be/ADtwyzyzNwA?si=xUBchREeEVvBL2ce

**Comment by themerekat** [comment]
Megan King suggests having a SSG about the loan management tools.

**Reply by ljwalker** [reply]
This topic has now been covered: https://youtu.be/RaeuXeKmFjE?si=q45WvfkdtP8sSjty

(Thanks for your suggestion, Megan King!)

**Comment by themerekat** [comment]
Suggestion from Mary Barkworth: "Make Using checklists as a topic. My impression is that they could be more widely used in building resources for teaching as well as research.  For example, if the only flora available for a region includes LOTS of species, use the tools in Symbiota to create a checklist of species for a smaller area and then make that the starting point for developing a dichotomous family key (which could be posted to KeyBase and the taxa linked to taxon pages in SEINet)."

**Reply by themerekat** [reply]
This has been covered! https://youtu.be/ADtwyzyzNwA?si=xUBchREeEVvBL2ce

**Comment by STHerbariumLSU** [comment]
It would be useful to have a session on the best workflow for a WeDigBio event using Symbiota.  Please include the best way to add records and review records from these events in "review/verify records".  Any tips to keep up with volunteer transcribers during an event would be helpful.  As well, the best way to clean up after an event.  This goes a bit outside of the Symbiota scope, but would be nice to have recorded, if not already!  Thanks! - Jennie Kluse

**Reply by ljwalker** [reply]
@STHerbariumLSU Thank you for making this suggestion!

While not WeDigBio-specific, you might find this Support Group meeting about crowdsourcing to be helpful: https://youtu.be/HmDMtRrLn-U?si=_ssWN7x7z_KXZkNj&t=263

**Comment by Alison-CenterforPlantDiversity** [comment]
Hello, I'm a new reviewer of student data entry, and I've figured out a few ways to quickly search for errors when QC'ing the entered data (four portals: CCH2, Bryoportal, Lichenportal and Macroalga Portal), but would like to learn more tricks for developing a checklist of what to check and how to see data needing editing. Thanks, Alison Colwell

**Reply by zoonomen-APP** [reply]
Hi Alison,
I am working on a project with Jim Hinds -- redoing the Lichen Checklist for New England. A great deal of my work (and contribution) consists of reviewing, editing, correcting the CLH held data for the related records. I live and breath daily by the joys and sorrows that result from the challenges of data entry. Despite my ranting and raving, my sympathies are with the data entry stalwarts.
I would be interested in hearing about your approaches and problems. I anticipate that my approach may differ substantially from yours but may still have a few tricks.
Alan P. Peterson
My email is appeterson37@gmail.com

**Comment by ljwalker** [comment]
Suggested during March 2024 SSG meeting: Best practices for including iNaturalist data in Symbiota portals. (@mickley!)

**Reply by mickley** [reply]
@ljwalker Sorry I missed this. I'll try to listen to the recording, but is there anything that I can add? 

I am also still waiting for my code that provides batch upload of linked resources from last March to be incorporated into Symbiota, see https://github.com/BioKIC/Symbiota/pull/701, so that is holding up any potential iNaturalist features on my end.

**Comment by ljwalker** [comment]
Suggested during March 2024: Label customization how-to (requested by @Alison-CenterforPlantDiversity & Karen Golinski)

**Reply by ljwalker** [reply]
This topic was originally covered in June 2022: https://youtu.be/EjydsbVGyRA?si=xPoMVT5qFlZlEVBC

**Reply by karenGolinski** [reply]
I watched the training some time ago and chatted with Katie about labels separately but I’m still not satisfied with the way they look. I will submit an email to discuss again. Thanks y’all.

**Comment by Alison-CenterforPlantDiversity** [comment]
Hi Alan,

OK I’ve worked down through the inbox to your first email of Sunday, now I’m getting the picture! Sounds like you have your hands full with the checklist of New England, but if you see other errors in DAV I’m happy to fix those for you. Please suggest what the correction likely is, though, to make it faster for me to figure out. If you’d like to do a session fixing DAV entry errors yourself, I’ll give you permission access.

We JUST rehoused and databased our entire lichen collection & bryo collection during the pandemic. I had a student – Karen Kyutoku - from Japan stuck here for a year after she graduated (couldn’t go home to Japan), and she did a beautiful job. A happy memory for me. She is now in a Master’s program in desert cryptogam ecology at Cal State LA. She will be on the job market soon so if you need a terrific curatorial assistant, I’ll put you in touch.

Other people doing checklists have been putting requests in the Symbiota “Comments” feature, which we do address weekly in our vascular portal (CCH2), but I’m pretty sure my collections manager does not check the comments in other portals, so I better go do that! Some folks (mainly my friends) get impatient waiting for comments to get addressed and email me directly with their requests. For example, Jerry Tiehm is doing the Flora of Nevada and he regularly emails me with “get this plant out of my flora!” We joke about that.

As for my tricks for catching errors, I’m going to have to look if I wrote down a protocol because I don’t remember them, need to circle back to that project…

Alison

**Comment by zoonomen-APP** [comment]
My hands are more than full.
BTW, I am NOT a lichenologist, or even a botanist.. I am a retired
physician (pathologist).
But I am accustomed to editing and handling moderately complex data (see my
website http://www.zoonomen.net).
Jim Hinds is an old friend for over 50 years and I was glad to help him on
his project.

Katie granted me data editing permission, quite a while ago. I use it
sparingly, mostly for obviously mis-transcribed data elements.
I enter a fair amount in Comments, some of these comments have been ignored
for almost 2 years.. and I get that.. there is limited resource to do all
that could be done.

I feel that the lichen collection/dataset is unusually important and
valuable for historical and environmental purposes, and in my view it is a
shame that more is not done to "tidy up" the data.
That said, I realize that academic advancement does not come from tidying
up datasets.

A

**Comment by Alison-CenterforPlantDiversity** [comment]
That is a very cool thing you are doing for your friend! Terrific!

I’m not looking for advancement, just a less frantic work week as I’m slowing down a bit. However, I do have volunteers in the lichen arena, and I could ask them to address comments or do data cleanup as their herbarium service. We are having a lichen keying session here in April, I will mention this then!

**Comment by zoonomen-APP** [comment]
Well it is pretty fun. It is what I am doing instead of slowing down.
For many understandable reasons, the Lichen data set is... well... pretty
untidy. With the mixture of amateur and professional collection, this is
not surprising.
And similarly (and understandably) data transcription has been seen
(wrongly) as essentially "scut work" with the main question being what is
the cheapest way of doing something here....

(Rant rave, Rant rave-- I could go on...)



Working with McCune is particular fun,,, he is quite a clever chap, and he
(largely) puts up with my foolishness.

Best,

A

**Comment by Alison-CenterforPlantDiversity** [comment]
Yes, well those of us who do museum curatorial work have that OCD thing to some extent, which makes us happy to organize databases but also unhappy with sloppiness. So a double edged sword!

Bruce McCune is a gem! He is an angel if the lichen community! Cherish working with him!

I have almost got to the lichen portal comments, but today we have some budget problems to solve, not nearly as fun.

Katelin Pearson told us on Monday about that “Lichen Portal Campaign” coming up in a month or two, that may be an opportunity to get help sorting out some of these things!

Alison

**Comment by themerekat** [comment]
Digitization workflows using Symbiota

**Comment by themerekat** [comment]
Mark Barkworth suggests how to use Symbiota portals to create floras/faunas as part of a research project. This uses checklists, but is more tailored toward how you can curate these over the course of a project.

**Comment by walterfertig** [comment]
A fun, interactive session might include techniques for georeferencing, such as using different map layers, employing Google map searches, and how to interpret labels like a field biologist. There may be some good hacks people have come up with across the network that they could share.

**Reply by themerekat** [reply]
@walterfertig , I could see that being interesting! That starts to slide a little bit away from the topic of Symbiota, but it's definitely of interest to the community.

**Comment by Alison-CenterforPlantDiversity** [comment]
Thank you, Walter!

I like this idea. The Geolocate is quirky enough that I can never remember how to use it (intermittent user) and definitely cannot properly train my people on it and I don't have time to sit through all four of their online trainings. The result is that unless a TRS is a simple section centroid, we skip the georeferencing step.

Time to improve the training kit on georeferencing!

________________________________

**Comment by walterfertig** [comment]
Hi Alison

Thanks for your interest.  Geolocate was built for mapping fish, so has a few quirks when applied to sessile organisms like plants. It is a useful tool to get you in the ballpark when mapping, but then it is best to use other info from the label to figure out the best spot to map something.  Georeferencing is also improved when the mapper has some knowledge of the organism being mapped (like its habitat preferences) and an understanding of how collectors work in the field and record data.  I have learned a few tricks over the years, both in the herbarium and from field work, to help train our student mappers.  Mapping is a skill of its own, and one we often are too quick to relegate to machines or un-trained workers to do.  Mismapped locations are the biggest source of error in Symbiota records, hands down.  I think it would be a relevant topic for the group.

Walter Fertig
Collections manager
Marion Ownbey Herbarium
Washington State University
Pullman, WA
________________________________
